models/baseModels/Payment/Payment.ts

```diff
@@ -445,6 +447,24 @@ export class Payment extends Transactional {
     throw new ValidationError(message);
   }
 
+  validatePartialAmount() {
+    if (!this.fyo.singles.AccountingSettings?.enablePartialPayment) {
+      const amount =
+        (this.amountPaid as Money)?.isZero?.() === false
```

Comment:
Here you don't have to explicitly check `something() === false`. `something() ? foo: bar ` would suffice.

Reply:
@suhailanzar 
Please mark a conversation as resolved only when it is resolved. I can still see the unchanged code. Please apply the mentioned fixes.